asv_bench/benchmarks/packers.py

```diff
@@ -547,6 +547,36 @@ def remove(self, f):
             pass
 
 
+class packers_write_json_lines(object):
+    goal_time = 0.2
+
+    def setup(self):
+        self.f = '__test__.msg'
+        self.N = 100000
+        self.C = 5
+        self.index = date_range('20000101', periods=self.N, freq='H')
```

Comment:
@joshowen you can leave it here as is. I cleaned this up in another PR (@jreback yes I know, I should merge that ...)